examples/milestone/TicTacToe/juvix.yaml

```
@@ -1,2 +1,3 @@
 name: TicTacToe
+main: TicTacToe.juvix
```

The main function is in `CLI/TicTacToe.juvix`, because we used to have CLI and Web versions. I assume paths of this type will work in this field?

```suggestion
main: CLI/TicTacToe.juvix
```